docs/EN_US/HPCCSpark/SparkHPCC.xml

```diff
@@ -432,12 +432,12 @@
                   xpointer="element(/1)"
                   xmlns:xi="http://www.w3.org/2001/XInclude" />
 
-      <para>Using a browser, navigate to your Integrated Master Spark instance
-      (the instance you added above) running on port 8080 of your HPCC
-      System.</para>
+      <para>Using a browser, navigate to your Integrated Manager Spark
```

Comment:
... navigate to your Integrated Spark's Manager instance...

Reply:
good catch